rest/src/routes/chainRoutes.js

```diff
@@ -20,25 +20,20 @@
 
 const routeResultTypes = require('./routeResultTypes');
 
-const curryStripProperties = (properties, next) => chainStatistic => {
-	properties.forEach(property => {
-		delete chainStatistic[property];
-	});
-
-	next(chainStatistic);
-};
-
-const currySend = (db, res, next) => chainStatistic => {
-	res.send({ payload: chainStatistic, type: routeResultTypes.chainStatisticCurrent });
-	next();
-};
-
 module.exports = {
 	register: (server, db) => {
-		server.get('/chain/height', (req, res, next) =>
-			db.chainStatisticCurrent().then(curryStripProperties(['scoreLow', 'scoreHigh'], currySend(db, res, next))));
-
-		server.get('/chain/score', (req, res, next) =>
-			db.chainStatisticCurrent().then(curryStripProperties(['height'], currySend(db, res, next))));
+		server.get('/chain/info', (req, res, next) =>
+			Promise.all([
+				db.chainStatisticCurrent(),
+				db.latestFinalizedBlock()
+			]).then(dbResults => {
+				const chainInfoResult = dbResults[0];
+				chainInfoResult.latestFinalizedBlock = dbResults[1].block;
```

Comment:
dbResults[1] could be undefined when starting the network. latestFinalizedBlock should be optional here and in the API. 
Also, once there is a value, 

finalizedHeight is not returned, I guess we can remove it from the open API as it's the same value than latestFinalizedBlock.height right?

Reply:
> dbResults[1] could be undefined when starting the network. latestFinalizedBlock should be optional here and in the API.
server will from now on always return something (i.e. the nemesis finalized block information), so this doesn't need to be changed in REST.